projects/transformers/run_args.py

```diff
@@ -321,6 +321,69 @@ class DataTrainingArguments:
                     "attempt to update existing entries"
         },
     )
+    max_train_samples: Optional[int] = field(
+        default=None,
+        metadata={
+            "help": "For debugging purposes or quicker training, truncate the number of"
+                    " training examples to this value if set."
+        },
+    )
+    max_eval_samples: Optional[int] = field(
+        default=None,
+        metadata={
+            "help": "For debugging purposes or quicker training, truncate the number of"
+                    " evaluation examples to this value if set."
+        },
+    )
+    max_predict_samples: Optional[int] = field(
+        default=None,
+        metadata={
+            "help": "For debugging purposes or quicker training, truncate the number of"
+                    " prediction examples to this value if set."
+        },
+    )
+    version_2_with_negative: bool = field(
+        default=False, metadata={"help": "If true, "
+                                         "some of the examples do not have an answer."}
+    )
+    null_score_diff_threshold: float = field(
+        default=0.0,
+        metadata={
+            "help": "The threshold used to select the null answer: if the best answer "
+                    "has a score that is less than the score of the null answer minus "
+                    "this threshold, the null answer is selected for this example. "
+                    "Only useful when `version_2_with_negative=True."
+        },
+    )
+    doc_stride: int = field(
```

Comment:
Adding the `squad_` prefix to these new squad arguments would help in separating namespaces.